src/code/softsprite_matrix_static.c

```diff
@@ -0,0 +1,10 @@
+#include "ultra64/types.h"
+
+//clang-format off
+Mtx gSoftSpriteMartixStatic = { 
+    65536 , 0     , 1     , 0     , 
+    0     , 65536 , 0     , 1     , 
+    0     , 0     , 0     , 0     , 
+    0     , 0     , 0     , 0     , 
+};
+//clang-format on
```

Comment:
what does it look like without clang-format off/on?

(it doesn't make more sense to format it as a 4x4 grid, the first 8 values are the integer parts, the last 8 the fractional parts)